sys-apps/shadow/shadow-4.4.ebuild

```diff
@@ -117,15 +131,14 @@ src_install() {
 	newins etc/login.defs login.defs
 	dosym ../usr/share/shadow/login.defs /etc/login.defs
 
+	set_login_opt CREATE_HOME yes
 	if ! use pam ; then
 		set_login_opt MAIL_CHECK_ENAB no
 		set_login_opt SU_WHEEL_ONLY yes
 		set_login_opt CRACKLIB_DICTPATH /usr/$(get_libdir)/cracklib_dict
 		set_login_opt LOGIN_RETRIES 3
 		set_login_opt ENCRYPT_METHOD SHA512
-
-		# CoreOS: increase the minimum password length to eight
-		set_login_opt PASS_MIN_LEN 8
```

Comment:
Note: this is a no-op change since we now use pam so this block never executes at all.